types/chrome/index.d.ts

```diff
@@ -3254,48 +3254,58 @@ declare namespace chrome.enterprise.platformKeys {
         /**
          * Which algorithm the registered key should use.
          */
-        algorithm: Algorithm;
+        algorithm: `${Algorithm}`;
     }
 
     /**
-     * @since Chrome 110
      * Type of key to generate.
+     * @since Chrome 110
      */
-    type Algorithm = "RSA" | "ECDSA";
+    export enum Algorithm {
+        ECDSA = "ECDSA",
+        RSA = "RSA",
```

Comment:
This does seem to be what the types mean more so than what it was, but have you been able to verify that the `Algorithm.RSA` works as expected in runtime? You don't really have to, it would just be nice (since I don't work on extensions anymore, I'd just rather make sure).

Reply:
hey @nicolas377, thank you for your review !

`Algorithm` and `Scope` exist in runtime 

A screenshot of my extension on Chome OS  pre-installed by policy : 
![image](https://github.com/user-attachments/assets/ee0f39cb-4380-4f54-bdda-84ea4a927f12)

I will also add screenshots for my other PRs :)